pkg/connector/sso_user.go

```diff
@@ -126,17 +125,17 @@ func ssoUserProfile(ctx context.Context, user awsIdentityStoreTypes.User) map[st
 	profile["aws_user_id"] = awsSdk.ToString(user.UserId)
 
 	if len(user.ExternalIds) >= 1 {
-		lv := &structpb.ListValue{}
+		lv := []interface{}{}
 		for _, ext := range user.ExternalIds {
-			attr, _ := structpb.NewStruct(map[string]interface{}{
+			attr := map[string]interface{}{
 				"id":     awsSdk.ToString(ext.Id),
 				"issuer": awsSdk.ToString(ext.Issuer),
-			})
+			}
 			if attr != nil {
```

Comment:
Like the linter says, you can remove this check because attr is never nil.

Reply:
Yep